website/react-magma-docs/src/pages/api/modal.mdx

```diff
@@ -197,6 +228,54 @@ export function Example() {
           <Button onClick={() => setShowModalHeader(false)}>OK</Button>
         </ButtonGroup>
       </Modal>
+
+      <Modal
+        onClose={() => setShowModalHeaderRef(false)}
+        isOpen={showModalHeaderRef}
+        headerRef={handleGetHeaderRef}
+        style={{ padding: '20px' }}
+      >
+        <Flex
+          behavior={FlexBehavior.container}
+          direction={FlexDirection.column}
+          alignItems={FlexAlignItems.center}
+          justify={FlexJustify.center}
+          spacing={2}
+          wrap={FlexWrap.nowrap}
+        >
+          <CheckIcon
+            style={{
+              background: magma.colors.success,
+              color: magma.colors.neutral100,
+              width: '36px',
+              height: '36px',
+              padding: '8px',
+              borderRadius: '50%',
+            }}
+          />
+          <h2 ref={customHeadingRef} tabIndex={-1}>
+            Add Links to Cengage Content
+          </h2>
+          <Paragraph>
+            Your LMS course is now integrated with Cengage. Select specific
+            Cengage activities to provide students with direct links. Or, add
+            just one link to your entire course. You can return to select more
+            content later.
+          </Paragraph>
```

Comment:
We'll keep the layout of this modal example, but we should generalize the content to not be Cengage specific (not that it's sensitive info, but just in case, lets not have this out in the public).